src/sql/workbench/parts/scripting/electron-browser/scripting.contribution.ts

```diff
@@ -83,7 +83,7 @@ MenuRegistry.appendMenuItem(MenuId.DataExplorerContext, {
 //#region -- Object Explorer
 
 MenuRegistry.appendMenuItem(MenuId.ObjectExplorerItemContext, {
-	group: 'scripting',
+	group: '0_query',
```

Comment:
I'm guessing the `0_` prefix has something to do with sort order?  In any case, it would be good to have a better solution\naming convention in the future.